include/tf-psa-crypto/build_info.h

```diff
@@ -143,23 +143,36 @@
 #endif
 #endif
 
-/* Auto-enable MBEDTLS_MD_C if needed by a module that didn't require it
- * in a previous release, to ensure backwards compatibility.
- */
-#if defined(MBEDTLS_PKCS5_C)
-#define MBEDTLS_MD_C
+/* Tweak the configuration of PSA mechanisms. */
+#include "tf-psa-crypto/private/crypto_adjust_config_synonyms.h"
+#include "tf-psa-crypto/private/crypto_adjust_config_auto_enabled.h"
+#include "tf-psa-crypto/private/crypto_adjust_config_dependencies.h"
+#include "tf-psa-crypto/private/crypto_adjust_config_key_pair_types.h"
+
+/* Define additional internal symbols based on the library configuration. */
+#include "tf-psa-crypto/private/crypto_adjust_config_derived.h"
+
+#if defined(MBEDTLS_PSA_CRYPTO_C)
+/* If we are implementing PSA crypto ourselves (as opposed to only
+ * having client-side stubs), enable built-in drivers for all the
+ * mechanisms activated with `PSA_WANT_xxx` that are not
+ * accelerated. */
+#include "mbedtls/private/crypto_adjust_config_enable_builtins.h"
+
+/* Special header to adjust the configuration to make a build
+ * where all enabled mechanisms are provided both as built-in and
+ * through libtestdriver1. See the comment at the top of the
```

Comment:
```suggestion
 * through test drivers. See the comment at the top of the
```
This is not used in `libtestdriver1` context I think.